examples/manipulation/grasp_eval.py

```diff
@@ -19,73 +22,234 @@
 import genesis as gs
 
 from grasp_env import GraspEnv
+from behavior_cloning import BehaviorCloning
+
+
+def load_rl_policy(env, train_cfg, log_dir):
+    """Load reinforcement learning policy."""
+    runner = OnPolicyRunner(env, train_cfg, log_dir, device=gs.device)
+
+    # Find the latest checkpoint
+    checkpoint_files = [f for f in os.listdir(log_dir) if re.match(r"model_\d+\.pt", f)]
+    if not checkpoint_files:
+        raise FileNotFoundError(f"No checkpoint files found in {log_dir}")
+
+    last_ckpt = sorted(checkpoint_files)[-1]
+    runner.load(os.path.join(log_dir, last_ckpt))
+    print(f"Loaded RL checkpoint from {last_ckpt}")
+
+    return runner.get_inference_policy(device=gs.device)
+
+
+def load_bc_policy(env, bc_cfg, log_dir):
+    """Load behavior cloning policy."""
+    # Create behavior cloning instance
+    bc_runner = BehaviorCloning(env, bc_cfg, None, device=gs.device)
+
+    # Find the latest checkpoint
+    checkpoint_files = [f for f in os.listdir(log_dir) if re.match(r"checkpoint_\d+\.pt", f)]
+    if not checkpoint_files:
+        raise FileNotFoundError(f"No checkpoint files found in {log_dir}")
+
+    last_ckpt = sorted(checkpoint_files)[-1]
+    last_ckpt_file = os.path.join(log_dir, last_ckpt)
+    print(f"Loaded BC checkpoint from {last_ckpt_file}")
+    bc_runner.load(last_ckpt_file)
+
+    return bc_runner._policy
+
+
+def get_stereo_frame(env, step_count):
+    """Get stereo frame as numpy array."""
+    # Get individual camera images
+    rgb_left, _, _, _ = env.left_cam.render(rgb=True, depth=False)
```

Comment:
You keep applying the same processing to right and left for this entire method and finally concatenate both image. Maybe you could store them in a list to avoid duplication?